### ISSUE TITLE
how to visualize the reconstruction result

### ISSUE THREAD

**Issue by YixuannnL** [phizaz/diffae#67]
Maybe a bit silly question is how you visualize your reconstruction result when your autoencoder training is done?
In my understanding maybe i should take the input to the decoder, which is the original image added noise, and using a noise scheduler denoise the input once. Just like
`latents = self.noise_scheduler.step(noise_pred, timesteps, dec_x).prev_sample`
`gen_img = self.image_decoder(latents)`
Is it reasonable?

Or should i denoise the noisy input of decoder iteratively in a for loop, using the noise predicted by the decoder? But i only add noise to the original image once, so i dont think this is reasonable maybe
